### PR TITLE
adds text to enchantress module

### PR DESCRIPTION
**Pull request by Jensen1989**
this is my first github edit, so I apologize if there's anything wrong

### CODE REVIEW

File: pack/mts_encounter.json [hunk 1]
```diff
@@ -1965,6 +1967,7 @@
 		"quantity": 2,
 		"set_code": "enchantress",
 		"set_position": 4,
+		"text": "Attach to your identity. You cannot make basic attacks or play attack events. Alter-Ego Action: spend [[energy]] [[mental]] resources -> discard this card",
```
Comment (lostie) [inline pack/mts_encounter.json:1970]: With missing formatting:

```suggestion
		"text": "Attach to your identity.\nYou cannot make basic attacks or play [[Attack]] events.\n<b>Alter-Ego Action</b>: Spend [energy] [mental] resources → discard this card.",
```

**Review by lostie — COMMENTED** [review]
I've left some comments about the formatting (for future reference it might be helpful to look at existing sets with the complete text to see examples of how to add similar formatted text).

Thanks for contributing! ;)

File: pack/mts_encounter.json [hunk 2]
```diff
@@ -1937,6 +1937,7 @@
 		"scheme": 2,
 		"set_code": "enchantress",
 		"set_position": 1,
+		"text": "When Revealed: Search the encounter deck, discard pile, and set-aside area for a copy of Seduced and attach it to your identity (Shuffle.)"
```
[the remark this answers is not in the document]
Comment (Jensen1989) [reply pack/mts_encounter.json:1940]: thank you for your great feedback, now that I feel a little more comfortable with what I'm doing, I will finish the rest of the mts_encounter text soon

**Comment by Jensen1989** [comment]
I added another pull request with all text updated

**Pull request closed without merging**